pkg/controllers/kepler_internal.go

```diff
@@ -149,6 +179,24 @@ func (r KeplerInternalReconciler) getInternal(ctx context.Context, req ctrl.Requ
 	return &ki, nil
 }
 
+func (r KeplerInternalReconciler) getRedfishSecret(secretName string) (*corev1.Secret, error) {
+	redfishSecret := corev1.Secret{}
+	redFishSecretName := types.NamespacedName{
+		Namespace: KeplerDeploymentNS,
```

Comment:
shouldn't this be the namespace that you get from the `kepler-internal` instead of the deployment namespace of `kepler` ? 
As a rule of thumb, internal must not use (or have any knowledge) of `Kepler` related config or variables.